sklearn/ensemble/_hist_gradient_boosting/grower.py

```diff
@@ -164,7 +164,7 @@ class TreeGrower:
     is_categorical : ndarray of bool of shape (n_features,), default=None
         Indicates categorical features.
     monotonic_cst : array-like of shape (n_features,), dtype=int, default=None
-        Indicates the monotonic constraint to enforce on each feature. -1, 1
+        Indicates the monotonic constraint to enforce on each feature. 1, -1
```

Comment:
Thanks for spotting this.

I think we might better address such documentation mistakes in `sklearn.ensemble._hist_gradient_boosting` doc-strings via another PR.

```suggestion
        Indicates the monotonic constraint to enforce on each feature. -1, 1
```